src/query/service/src/interpreters/interpreter_table_show_create.rs

```diff
@@ -71,8 +71,8 @@ impl Interpreter for ShowCreateTableInterpreter {
             for (idx, field) in schema.fields().iter().enumerate() {
                 let default_expr = match field.default_expr() {
                     Some(expr) => {
-                        let expression = PlanParser::parse_expr(expr)?;
-                        format!(" DEFAULT {}", expression.column_name())
+                        let expression: PhysicalScalar = serde_json::from_str(expr)?;
```

Comment:
Is it possible to use `PhysicalScalar` as `DataFiled::default_expr`

Reply:
I have tried it but seems to need a nontrivial change